options/locale/locale_en-US.ini

```diff
@@ -3679,6 +3679,7 @@ runs.no_workflows.quick_start = Don't know how to start with Gitea Actions? See
 runs.no_workflows.documentation = For more information on Gitea Actions, see <a target="_blank" rel="noopener noreferrer" href="%s">the documentation</a>.
 runs.no_runs = The workflow has no runs yet.
 runs.empty_commit_message = (empty commit message)
+runs.expire_log_message = Logs have been cleaned up because they were too old. You can configure LOG_RETENTION_DAYS to retain logs for a longer duration.
```

Comment:
```suggestion
runs.expire_log_message = Logs have been purged because they were too old. You can configure LOG_RETENTION_DAYS to retain logs for a longer duration.
```